### PR TITLE
Generate  types

### PR DESCRIPTION
**Pull request by yukukotani**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "Ubie Design Tokens",
   "style": "dist/tokens.css",
   "main": "dist/tokens.js",
-  "typings": "design-tokens.d.ts",
+  "typings": "dist/tokens.d.ts",
   "files": [
     "dist/**",
     "design-tokens.d.ts"
```
Comment (takanorip) [inline package.json]: Please delete this line
@yukukotani

Comment (yukukotani) [reply package.json]: @takanorip nice catch! [395c491](https://github.com/ubie-oss/design-tokens/pull/10/commits/395c491b8ac1b86dc633bda0be213a889af4fe14)

**Comment by takanorip** [comment]
Please bump up version of package.json to `v0.0.3` in this PR and push git tag `v0.0.3` after this PR is merged 🙏 
@yukukotani

**Comment by yukukotani** [comment]
@takanorip Ok, done

**Review by takanorip — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**